.github/workflows/fidelity-tests.yml

```diff
@@ -37,8 +37,13 @@ jobs:
 
     - name: Build packages
```

Comment:
Is it ok if i change this to build only model-viewer package and fidelity test package? As there are more and more packages, this can save quite some time

Reply:
Sure, though I'm pretty sure you'll also need to build 3dom, as that's a dependency of model-viewer.